tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -37,6 +39,11 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |}
       |""".stripMargin,
     "",
+    compat = Map(
+      "3" -> """|transient scala (commit: '')
```

Comment:
That's inside a comment, so we should not show anything here :thinking:

Reply:
Agreed but this was broken all along. Just the test did not test it properly. Let's maybe do it as a followup.

Reply:
Ach, why did the test not show it before? Just because there were fewer than 3 letters?

Reply:
Yes, it only searched for exact matches in workspace/classpath when less than 3 chars.

Reply:
ok sure, let's create an issue then

Reply:
https://github.com/scalameta/metals/issues/5971